content/desktop/hardened-desktop/settings-management/configure.md

```diff
@@ -166,6 +189,9 @@ The following `admin-settings.json` code and table provides an example of the re
 | `allowExperimentalFeatures`| | If `value` is set to `false`, experimental features are disabled.|
 | `allowBetaFeatures`| | If `value` is set to `false`, beta features are disabled.|
 | `blockDockerLoad` | | If `value` is set to `true`, users are no longer able to run [`docker load`](../../../engine/reference/commandline/image_load.md) and receive an error if they try to.|
+| `filesharingAllowedDirectories` |  | Specify which paths your developers can add file shares to. Also accepts `$HOME`, `$TMP`, or `$TEMP` as `path` variables. When a path is added, its subdirectories are allowed. If `sharedByDefault` is set to `true`, that path will be added upon factory reset or when Docker Desktop first starts. |
+| `useVirtualizationFrameworkVirtioFS`|  <span class="badge badge-info">macOS only</span> | If `value` is set to `true`, VirtioFS is set as the file sharing mechanism. Note: If both `useVirtualizationFrameworkVirtioFS` and `useGrpcfuse` have `value` set to `true`, VirtioFS takes precedence. Likewise, if both `useVirtualizationFrameworkVirtioFS` and `useGrpcfuse` have `value` set to `false`, osxfs is set as the file sharing mechanism. |
+| `useGrpcfuse` | <span class="badge badge-info">macOS only</span> | If `value` is set to `true`, gRPC Fuse is set as the file sharing mechanism. |
```

Comment:
the `span` elements don't do anything in this table so I suggest we remove them, or if you want to have badges you can use the `badge` shortcode